tests/Unit/Css/CssDocumentTest.php

```diff
@@ -570,16 +570,20 @@ public function provideInvalidNonConditionalAtRuleWhichCausesException(): array
     public function provideInvalidNonConditionalAtRuleWhichDoesNotCauseException(): array
     {
         return [
-            '`@font-face` without `font-family`' => ['
+            '`@font-face` without `font-family`' => [
+                '
                 @font-face {
                   src: url("/foo-sans.woff2") format("woff2");
                 }
-            '],
-            '`@font-face` without `src`' => ['
+            ',
+            ],
+            '`@font-face` without `src`' => [
+                '
                 @font-face {
                   font-family: "Foo Sans";
                 }
-            '],
+            ',
+            ],
```

Comment:
The opening and trailing whitespace in the string was just for readability.  If we want the opening and closing quote on separate lines from square brackets (which I'm fine with), we can add some more whitespace so that readability is maintained, e.g.
```php
            '`@font-face` without `font-family`' => [
                '
                    @font-face {
                      src: url("/foo-sans.woff2") format("woff2");
                    }
                ',
            ],
            '`@font-face` without `src`' => [
                '
                    @font-face {
                      font-family: "Foo Sans";
                    }
                ',
            ],
```

Reply:
Done.